### ISSUE TITLE
Unable to deploy vm's without trusted launch.

### ISSUE THREAD

**Issue by nandosie012** [Azure/AVDSessionHostReplacer#6]
Why am i not able to deploy vm's without the trusted launch option using powershell,

Tried editting the parameters in de tempaltespecs file and also in the confguration my function app but unfortunately the only way for me to do this will be using trusted launch and not enabeling the vtpm and secure boot options. this works but that means a bunch of image rewriting..

**Comment by ankurvvvv** [comment]
Same issue. My images are build with Standard boot (No Trusted or confidentials boot).

When i hit review & create, i got this error:

{"code":"InvalidTemplate","message":"Deployment template validation failed: 'The value for the template parameter 'SecureBootEnabled' at line '94' and column '26' is not provided. Please see https://aka.ms/arm-create-parameter-file for usage details.'."}

**Comment by joshbasquez** [comment]
@ankurvvvv  @nandosie012  -- are you both using portal-ui deployment from this repo? I saw that it returns the value of the checkboxes but those are hidden when standard mode is selected. Could be fixed there but I was able to fix by setting a default value in the file:
deploy\arm\DeployAVDSessionHostReplacer.json [arm template]. 

I added "defaultValue": false around line 98 for securebootenabled and tpmenabled. 

`"SecureBootEnabled": {
      "type": "bool",
      "defaultValue": false
    },
    "TpmEnabled": {
      "type": "bool",
      "defaultValue": false
    },`

I'll reach out to Willy
